### ISSUE TITLE
RuntimeError: Error, unable to determine correct filename for 64bit macos

### ISSUE THREAD

**Issue by bencherkihamza** [InstaPy/InstaPy#6159]
I installed Instapy but I still get this error message : 'RuntimeError: Error, unable to determine correct filename for 64bit macos' even though I applied the solutions offered on issues # 5672 and # 5282. Could you help me please ?
<img width="1004" alt="Capture d’écran 2021-04-19 à 13 19 01" src="https://user-images.githubusercontent.com/23017379/115246823-c10a6080-a115-11eb-9dd4-1b0a32af8863.png">

**Comment by mian-amir** [comment]
need help sir  facebook commonds

**Comment by bencherkihamza** [comment]
> need help sir facebook commonds

Sorry, i don't get it

**Comment by leonardozhe** [comment]
> I installed Instapy but I still get this error message : 'RuntimeError: Error, unable to determine correct filename for 64bit macos' even though I applied the solutions offered on issues # 5672 and # 5282. Could you help me please ?
> <img alt="Capture d’écran 2021-04-19 à 13 19 01" width="1004" src="https://user-images.githubusercontent.com/23017379/115246823-c10a6080-a115-11eb-9dd4-1b0a32af8863.png">

try sudo apt install firefox-geckodriver
i can fix above issue by add firefox geckodriver.

**Comment by 4enterthedragon7** [comment]
same here guys

**Comment by amerikanets** [comment]
I believe the issue is with geckodriver now having 2 versions of the file for mac os:

<img width="985" alt="Screen Shot 2021-04-24 at 12 32 23 pm" src="https://user-images.githubusercontent.com/83103654/115944452-2fc23200-a4f9-11eb-88f1-eaaf5275398d.png">

What seems to work for me is updating webdriverdownloader.py file as follows:

Line 320 to: 

`filename = [name for name in filenames if os_name in name and name[-12:] == 'macos.tar.gz' ]`

Basically filtering the version of the geckodriver that we need.

**Comment by kaerus01** [comment]
I'm seeing this same issue I modified line 320 to the above with `name[-12:] == 'macos.tar.gz'` and have the geckodriver installed via npm

```
$ npm ls
webdriverdownloader@ /Library/Python/3.7/site-packages/webdriverdownloader
└── geckodriver@1.22.3
```

I'm still seeing this error:

```
File "/Library/Python/3.7/site-packages/webdriverdownloader/webdriverdownloader.py", line 177, in download_and_install
    show_progress_bar=show_progress_bar)
  File "/Library/Python/3.7/site-packages/webdriverdownloader/webdriverdownloader.py", line 129, in download
    download_url = self.get_download_url(version, os_name=os_name, bitness=bitness)
  File "/Library/Python/3.7/site-packages/webdriverdownloader/webdriverdownloader.py", line 324, in get_download_url
    raise RuntimeError(info_message)
RuntimeError: Error, unable to determine correct filename for 64bit macos
```
